src/components/chips/chips.js

```diff
@@ -44,39 +42,40 @@ const chipItem = css`
     padding-left: 4px;
     vertical-align: middle;
     &:hover {
-      color: ${$hotCoral};
+      color: #ef4056;
     }
   }
 
   &:hover {
-    border-color: ${$overcast};
+    border-color: #999;
   }
 `
 
 const chipItemActive = css`
-  background-color: ${$smoke};
-  padding: '2px 5px 2px 6px';
+  background-color: #ddd;
+  padding: 2px 5px 2px 6px;
```

Comment:
Updated the styles here to have the right colors, still needs some darkMode adjustments and some final tweaks. Will be much easier once storybooks is in place.

Reply:
Love this. I think this is a much better starting place.